packages/storybook-vue/stories/components/icon/Icon.stories.mdx

```diff
@@ -46,10 +46,7 @@ export const PATH =
       control: {
         type: 'text',
       },
-    },
-    styles: {
-      control: { disable: true },
-    },
+    }
   }}
```

Comment:
add the style here again since without it the component is not working correctly